### ISSUE TITLE
Clarify period for framesDropped / FramesLost

### ISSUE THREAD

**Issue by stefhak** [w3c/webrtc-stats#458]
The spec text talks about `total number of frames` and `cumulative number of partial/full frames`; total/cumulative hints measuring period would be 'since receiver was created', however RFC7004 appendix A is referenced, and RFC7004 references RFC3611 for `begin_seq` and `end_seq`.

I think this should be clarified.

**Comment by vr000m** [comment]
We should reference Sections A.g, h, i, j..

Basically,
```
 Number of duplicate frames (dup_frames): 32 bits

      Number of duplicate frames received in the above sequence number
      interval.

   Number of full lost frames (full_lost_frames): 32 bits

      A frame is either split across multiple packets or carried in only
      one packet.  If the whole frame or all the packets of the frame
      are lost, this frame is regarded as one full_lost_frame.  The
      full_lost_frames can be inferred from packet(s) that comprise the
      frame.  The full_lost_frames is equivalent to the number of full
      lost frames in the above sequence number interval.

   Number of partial lost frames (partial_lost_frames): 32 bits

      When a frame is split across multiple packets and some packets of
      the frame are lost, this frame is regarded as one
      partial_lost_frame.  The partial_lost_frames can be inferred from
      packets that comprise the frame.  The value of the
      partial_lost_frames field is equivalent to the number of partial
      lost frames in the above sequence number interval.
``

**Comment by vr000m** [comment]
In RTP the frames can be identified from the fragemented packets.
All packets have the same RTP Timestamp, they should be contiguous sequence numbers, the last packet in the frame has the market bit set. 

IIRC, the begin_seq and end_seq in RFC7004 are more important for the other metrics being reported in the packet (namely burst/gap metrics).

**Comment by stefhak** [comment]
I did parse the definitions, but I find them hard to use since I don't know what `the above sequence number interval` is. Basically I had expected x_lost_frames to be stable or increasing over a session but was surprised to find out that they can go up and down (e.g. getting a sequence like 1, 0, 1, 0 , 2, ... when sampling every n seconds) during a session when I did a small test.

**Comment by vr000m** [comment]
I agree that we should make clear that this is not an interval metric but the start from the appearance of the Receiver.

@stefhak: For the XR in RFC7004 what was the setting for the I (interval flag).

```
This field is used to indicate whether the burst/gap loss summary
      statistics metrics are Sampled, Interval, or Cumulative metrics:

         I=10: Interval Duration - the reported value applies to the
         most recent measurement interval duration between successive
         metrics reports.

         I=11: Cumulative Duration - the reported value applies to the
         accumulation period characteristic of cumulative measurements.

         I=01: Sampled Value - the reported value is a sampled
         instantaneous value.
```

**Comment by henbos** [comment]
Fixed by #490
